app/controllers/tasks_controller.rb

```diff
@@ -28,8 +28,21 @@ def create
 
   def edit
     @task = Task.find(params[:id])
-    authorize @project
+    @project = Project.find(params[:project_id])
```

Comment:
We can make the controller more DRY by defining a private method with find(params)